release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -163,6 +163,13 @@ For more information, see xref:../networking/ovn_kubernetes_network_provider/tra
 
 For more information, see xref:../networking/ingress-operator.adoc#nw-ingress-converting-http-header-case_configuring-ingress[Converting HTTP header case].
 
+[id="ocp-4-8-ingress-configuring-proxy-protocol"]
+==== Configuring PROXY protocol for an Ingress Controller
+
+{product-title} 4.8 adds support for configuring PROXY protocol for the Ingress Controller API as a way to enhance communication between the Ingress Controller and an external load balancer.
```

Comment:
Should references to "PROXY protocol" have a "the" in front ("the PROXY protocol")?

Reply:
Yep! http://www.haproxy.org/download/2.2/doc/proxy-protocol.txt